mcc-wrapper/src/main/java/de/verdox/mccreativelab/wrapper/world/level/biome/MCCBiome.java

```diff
@@ -0,0 +1,49 @@
+package de.verdox.mccreativelab.wrapper.world.level.biome;
+
+/**
+ * Represents a MCC Biome
+ */
+public interface MCCBiome {
+
+    /**
+     * Checks if the biome has precipitation
+     *
+     * @return true if the biome has precipitation
+     */
+    Boolean hasPrecipitation();
+
+    /**
+     * Gets the base temperature of the biome
+     *
+     * @return the base temperature
+     */
+    float getBaseTemperature();
+
+    /**
+     * Gets the temperature modifier of the biome
+     *
+     * @return the temperature modifier
+     */
+    MCCBiomeSpecialEffects getSpecialEffects();
+
+    /**
+     * Enum for temperature modifiers
+     */
+    enum MCCTemperatureModifier {
```

Comment:
We don't need the 'MCC' prefix for inner classes.

Change MCCTemperatureModifier -> TemperatureModifier.

Reply:
Thank you! I changed it [here](https://github.com/derverdox/MCCreativeLab/pull/10/commits/19e07bc947a2c968810affc5d4d512d769304c13).